raiden_contracts/utils/versions.py

```diff
@@ -16,19 +13,20 @@ def contracts_version_expects_deposit_limits(contracts_version: Optional[str]) -
 
 
 def contract_version_with_max_token_networks(version: Optional[str]) -> bool:
-    manager = ContractManager(contracts_precompiled_path(version))
-    abi = manager.get_contract_abi(CONTRACT_TOKEN_NETWORK_REGISTRY)
-    constructors = list(filter(lambda x: x["type"] == "constructor", abi))
-    assert len(constructors) == 1
-    inputs = constructors[0]["inputs"]
-    max_token_networks_args = list(filter(lambda x: x["name"] == "_max_token_networks", inputs))
-    found_args = len(max_token_networks_args)
-    if found_args == 0:
+    if version is None:
+        return True
+    if version == "0.3._":
+        return False
+    if version == "0.8.0_unlimited":
+        return False
+    return compare(version, "0.9.0") >= 0
+
+
+def contracts_version_provides_services(version: Optional[str]) -> bool:
+    if version is None:
+        return True
```

Comment:
Why would a None-versioned contract provide services?

Reply:
None is the cutting-edge, not-yet-versioned version.

Reply:
But I think I'll remove the None version from the PyPI distribution https://github.com/raiden-network/raiden-contracts/issues/1036 .

Reply:
Sounds good